src/client/components/notFound/NotFound.module.css

```diff
@@ -0,0 +1,18 @@
+.not-found {
+  background-color: #232323;
+  width: 1200px;
+  height: 561px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.not-found h1 {
```

Comment:
let's instead of using tag create a classname for h1

Reply:
Corrected